STRUCTURE.md

```diff
@@ -17,15 +24,15 @@ documentation of use.
 
 Specifically, the following principles apply:
 
-* Users of these libraries should never have to even think about how the
+* Users of these libraries SHOULD never have to even think about how the
   libraries are bundled or linked to. Any and all handling of this, for any
-  platform, must be done by the library.
-* These libraries cannot have any dependencies other than `base`.
-* The documentation for the modules must be strong enough to stand alone:
-  specifically, at least 80% of all questions regarding the use of bindings must
+  platform, MUST be done by the library.
+* These libraries SHOULD NOT have any dependencies other than `base`.
```

Comment:
I feel this isn't strong enough. SHOULD NOT means that we can bend the rules and depend on something other than `base`, which for the C-esque level is unwarranted IMHO.

Reply:
It means that we can bend the rules provided that the full implications must be understood and carefully weighed before choosing a different course. From what I understand, this allows us to enforce it while giving us some leeway should the occasion arise that we need something that ought to be in base but isn't yet (like the case `primitive`, for instance).

Reply:
I can't think of a situation where _bindings_ would require `primitive`.